.github/workflows/netlify.yaml

```diff
@@ -33,28 +33,8 @@ jobs:
                   });
                   var fs = require('fs');
                   fs.writeFileSync('${{github.workspace}}/previewbuild.zip', Buffer.from(download.data));
-
-                  var prInfoArtifact = artifacts.data.artifacts.filter((artifact) => {
-                    return artifact.name == "pr.json"
-                  })[0];
```

Comment:
you should probably stop creating the `pr.json` artifact if you're not using it any more?

Reply:
Oops, yep - thanks!